django/cantusdb_project/main_app/tests/test_models.py

```diff
@@ -90,7 +90,7 @@ def test_index_components(self):
         self.assertEqual(weight_search_term_dict, expected_dict)
 
     def test_get_concordances(self):
-        chant = Chant.objects.get(id=1)
+        chant = Chant.objects.first()
```

Comment:
Not part of this pr?

Reply:
I'm not sure why this particular test chose now to begin failing, but it did. I don't know why it had been set up differently from other tests in the test suite, but bringing it into alignment with the others causes it to behave as expected.

I can revert this and open a separate PR if you think it's worth doing.

Reply:
Nah, just pointing it out in case it slipped in.